pyproject.toml

```diff
@@ -37,16 +44,6 @@ lint = ["ruff", "black"]
 test = ["pytest"]
 dev = ["ruff", "black", "pytest", "tyro", "pandas", "pyright"]
 
-[tool.setuptools]
-include-package-data = false
-
-[tool.setuptools.dynamic]
-version = {attr = "lowtime.__version__"}
```

Comment:
I am not quite sure what you mean (my knowledge on packaging/distribution is still kind of weak). I can see that `lowtime/__init__.py` specifies a version. I can see that we used to dynamically parse this version to set some (I'm guessing required) `setuptools` field. Since we change the `build-backend` to use `setuptools` instead, I'm not sure what `maturin` needs us to specify... Could you rephrase what you mean?

Reply:
Maturin currently does not support dynamic versions (https://github.com/PyO3/maturin/issues/199). Let's hardcode the version of lowtime to a specific number for now. When we release, we want to ensure that the version string of `lowtime.__version__`, Cargo.toml, and pyproject.toml are the same.